homeassistant/data_entry_flow.py

```diff
@@ -268,7 +268,7 @@ class FlowHandler:
     flow_id: str = None  # type: ignore
     hass: HomeAssistant = None  # type: ignore
     handler: str = None  # type: ignore
-    context: Dict = None  # type: ignore
+    context: Dict = {}
```

Comment:
We don't want to initialize the class attribute with a mutable object. Then updating the context will update all instance of this class. We'll overwrite this attribute after creating the handler instance but it's a not needed risk.

Reply:
Should not be possible, but still a fair point. Made sure it has a subscriptable, but immutable, default.